druntime/src/core/thread/osthread.d

```diff
@@ -1707,6 +1707,10 @@ private extern (D) bool suspend( Thread t ) nothrow @nogc
  * Throws:
  *  ThreadError if the suspend operation fails for a running thread.
  */
+extern (C) void thread_preStopTheWorld() nothrow {
```

Comment:
This probably shouldn't have been inserted in the middle of `thread_suspendAll`'s ddoc comment and implementation

Reply:
Fair enough, I moved the code and added a doc comment for it.